notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<ol><li>How about "That's why we use weight compression" -&gt; "That's why we use <em>only</em> weight compression" ? </li><li><code>vae encoder and decoder</code> -&gt; <code>vae_encoder</code> and <code>vae_decoder</code></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>